### ISSUE TITLE
Bug: Network/Aggregate is recommending offline encoders.

### ISSUE THREAD

**Issue by republicus** [YouPHPTube/YouPHPTube-Encoder-Network#3]
The Network/Aggregate is recommending offline encoders.

![phptube-network](https://user-images.githubusercontent.com/47370709/59402475-657a2980-8d6c-11e9-8bea-4ece8f20c7b5.PNG)

I suspect a failed ping is treated as a **0 ms** reply ; which would rank the offline encoder higher than the other encoders since a lower ping is desirable.

Steps to reproduce: 
edit /etc/hosts; point encoder to broken IP.
[Observe that restoring /etc/hosts entry to a good IP also restores proper aggregate recommendation.]

Results:
Offline encoder becomes the recommended encoder.

Expected results:
Offline encoder is never recommended.

**Comment by akhilleusuggo** [comment]
@republicus Yes , this bug is already reported multiple times and looks like they can't fix it . 
In your case your site is down , not that the encoder is failing the ping

**Comment by DanielnetoDotCom** [comment]
Thanks, this bug is on out to do list
